main.py

```diff
@@ -2,6 +2,58 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    # create
+    genres_names_to_create = ["Western", "Action", "Dramma"]
+    for genre_name in genres_names_to_create:
+        Genre.objects.create(
+            name=genre_name
+        )
+
+    actors_names_to_create = [
+        "George Klooney", "Kianu Reaves", "Scarlett Keegan",
+        "Will Smith", "Jaden Smith", "Scarlett Johansson"
+    ]
+    for actor_name in actors_names_to_create:
+        actor_name_dict = {"first_name": actor_name.split()[0],
+                           "last_name": actor_name.split()[1]}
+        Actor.objects.create(**actor_name_dict)
+
+    # update
+    Genre.objects.filter(name="Dramma").update(name="Drama")
+
+    # a - automatization
+    actors_to_update = {
+        "names_to_update": [
+            "George Klooney", "Kianu Reaves"
+        ],
+        "new_atrbs": [
+            {
+                "last_name": "Clooney",
+            },
+            {
+                "first_name": "Keanu", "last_name": "Reeves"
+            }
+        ]
+    }
+    for index in range(len(actors_to_update["names_to_update"])):
+        actor_name_dict = {
+            "first_name":
+                actors_to_update["names_to_update"][index].split()[0],
+            "last_name":
+                actors_to_update["names_to_update"][index].split()[1]
+        }
+        Actor.objects.filter(
+            **actor_name_dict
+        ).update(
+            **actors_to_update["new_atrbs"][index]
+        )
```

Comment:
Just do it by single names . We do not need dict fot updating data here

Reply:
i just wanted to implement more auto process for updating, but I agree that was horrible)